syft/workers/base.py

```diff
@@ -217,12 +217,12 @@ def send_msg(self, msg_type: int, message: str, location: "BaseWorker") -> objec
             location.
         """
         if self.verbose:
-            print(f"worker {self} sending {msg_type} {message} to {location}")
+            print(f"worker {self} sending {message} to {location}")
+        #
+        # # Step 0: combine type and message
+        # message = messaging.Message(msg_type, message)
```

Comment:
why is this commented out?

Reply:
Oh - i removed that step - i can delete it now.

Reply:
fixed